frontend/packages/topology/src/utils/moveNodeToGroup.tsx

```diff
@@ -1,21 +1,31 @@
 import * as React from 'react';
+import { TFunction } from 'i18next';
 import { confirmModal, errorModal } from '@console/internal/components/modals';
 import { Node } from '@patternfly/react-topology';
 import { updateTopologyResourceApplication } from './topology-utils';
 
-export const moveNodeToGroup = (node: Node, targetGroup: Node): Promise<void> => {
+export const moveNodeToGroup = (t: TFunction, node: Node, targetGroup: Node): Promise<void> => {
   const sourceGroup = node.getParent() !== node.getGraph() ? (node.getParent() as Node) : undefined;
   if (sourceGroup === targetGroup) {
     return Promise.reject();
   }
 
   if (sourceGroup) {
-    const title = targetGroup ? 'Move Component Node' : 'Remove Component Node from Application';
+    const title = targetGroup
+      ? t('topology~Move Component Node')
+      : t('topology~Remove Component Node from Application');
     const message = (
       <>
-        Are you sure you want to {targetGroup ? 'move' : 'remove'}{' '}
-        <strong>{node.getLabel()}</strong> from {sourceGroup.getLabel()}
-        {targetGroup ? ` to ${targetGroup.getLabel()}` : ''}?
+        {targetGroup
+          ? t('topology~Are you sure you want to move ')
+          : t('topology~Are you sure you want to remove ')}
+        <strong>{node.getLabel()}</strong>
+        {targetGroup
+          ? t('topology~ from {{sourceGroup}} to  {{targetGroup}}?', {
+              sourceGroup: sourceGroup.getLabel(),
+              targetGroup: targetGroup.getLabel(),
+            })
+          : t('topology~ from {{sourceGroup}}?', { sourceGroup: sourceGroup.getLabel() })}
```

Comment:
🤔  There is a pattern that can be used here...

Quick search through the codebase got me: 
```
"teams-help": "Optionally list organizations. If specified, only GitHub users that are members of at least one of the listed organizations will be allowed to log in. Cannot be used in combination with <1>teams</1>.",
```
And:
```
            <Trans i18nKey="teams-help" ns="github-idp-form">
              Optionally list organizations. If specified, only GitHub users that are members of at
              least one of the listed organizations will be allowed to log in. Cannot be used in
              combination with <strong>teams</strong>.
            </Trans>
```
Ref: [github-idp-form.tsx](https://github.com/openshift/console/blob/master/frontend/public/components/cluster-settings/github-idp-form.tsx#L228-L232)

cc @christianvogt might know more -- I'm reverse engineering here.